backends/ebpf/README.md

```diff
@@ -1,17 +1,27 @@
 # eBPF Backend
 
-The back-end accepts only P4_16 code written for the `ebpf_model.p4`
-filter model.  It generates C code that can be afterwards compiled
-into eBPF (extended Berkeley Packet Filters
-https://en.wikipedia.org/wiki/Berkeley_Packet_Filter) using clang/llvm
-or bcc (https://github.com/iovisor/bcc.git).
+The back-end accepts only P4_16 code written for the `ebpf_model.p4` or
+`xdp_model.p4` filter models.  It generates C code that can be afterwards
```

Comment:
Is this accurate? I do not think this works unless we integrate https://github.com/vmware-archive/p4c-xdp?

Reply:
The PR #4160 effectively did that. See for example this test case: https://github.com/p4lang/p4c/blob/main/testdata/p4_16_samples/xdp_dec_ttl_ebpf-kernel.p4